polar-core/src/error.rs

```diff
@@ -426,8 +426,8 @@ impl fmt::Display for ValidationError {
             Self::InvalidRule { rule, msg } => {
                 write!(f, "Invalid rule: {} {}", rule, msg)
             }
-            Self::InvalidPrototype { prototype, msg } => {
-                write!(f, "Invalid prototype: {} {}", prototype, msg)
+            Self::InvalidRuleType { rule_type, msg } => {
+                write!(f, "Invalid rule_type: {} {}", rule_type, msg)
```

Comment:
Can you check to see if this happened other places too?
```suggestion
                write!(f, "Invalid rule type: {} {}", rule_type, msg)
```

Reply:
Yep, found and fixed one other.